<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 42, 54)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame_top">
      <property name="minimumSize">
       <size>
        <width>1920</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(68, 71, 90)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_side">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>1080</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(68, 71, 90)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="frame_menu">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>1080</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(68, 71, 90)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_account">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Nirmala UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
            </property>
            <property name="text">
             <string>Account</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_harian">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Nirmala UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
            </property>
            <property name="text">
             <string>Kasus Harian COVID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pesan">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Nirmala UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(68, 71, 90);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 121, 198);
}</string>
            </property>
            <property name="text">
             <string>Pesanan Rumah Sakit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_logout">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Nirmala UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 85, 85);
	border: 0px solid;
	text-align: left;
	padding: 10px
}
QPushButton:hover {
	background-color:rgb(255, 100, 115);
}</string>
            </property>
            <property name="text">
             <string>Log Out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>138</width>
               <height>397</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stack_widget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_account">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QFrame" name="frame_top_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../img/nopicture.png</pixmap>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="lbl_info">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>150</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Nirmala UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(139, 233, 253)</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_bottom">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
